x/meicdp/client/cli/query.go

```diff
@@ -9,23 +9,24 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/spf13/cobra"
 
-	"github.com/bandprotocol/band-consumer/x/consuming/types"
+	"github.com/tharamalai/meichain/x/meicdp/types"
 )
 
 // GetQueryCmd returns
 func GetQueryCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
-	consumingCmd := &cobra.Command{
+	meiCdpCmd := &cobra.Command{
 		Use:                        types.ModuleName,
-		Short:                      "Querying commands for the consuming module",
+		Short:                      "Querying commands for the meicdp module",
 		DisableFlagParsing:         true,
 		SuggestionsMinimumDistance: 2,
 		RunE:                       client.ValidateCmd,
 	}
-	consumingCmd.AddCommand(flags.GetCommands(
+	meiCdpCmd.AddCommand(flags.GetCommands(
 		GetCmdReadResult(storeKey, cdc),
```

Comment:
Still using GetCMDReadResult?

Reply:
Only for testing, will remove after we don't use it.